.travis.yml

```diff
@@ -1,7 +1,6 @@
 ---
 language: python
 python:
-  - "2.7"
   - "3.5"
```

Comment:
```suggestion
```
Can we drop 3.5 as well?

Reply:
Let's do it. Probably doesn't add much value anyway.